.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
```

Comment:
In the .editorconfig file the `indent_size` is 4, here the `tabWidth` is 2. Is this on purpose?